assets/js/auto-anchar.js

```diff
@@ -0,0 +1,24 @@
+function addAnchorLink( heading ) {
+    var link = heading.innerText;
+    var linkToLowerCase = link.toLowerCase().replace(/\s+/g, '-').replace(/[&\/\\#,^!+()$~%.\[\]'":*?;-_<>{}@‘’”“|]/g, '-');
+    heading.innerHTML = '\n\t\t<a href="#'
+        .concat( linkToLowerCase, '" id="' )
+        .concat( linkToLowerCase, '" class="bsf-changelog-anchors">\n\t\t\t<i class="dashicons dashicons-paperclip"></i>\n\t\t\t' )
+        .concat( heading.innerHTML, '\n\t\t</a>\n\t' );
+}
+
+function bsfChangelogScrollToView() {
+    var hash = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : "";
+    if ( "" == hash ) return;
+    element = document.querySelector(hash);
+    if ( null === element ) return;
+    element.scrollIntoView({ behavior: "smooth", block: "start" });
```

Comment:
element.scrollIntoView is not supported in IE and safari, what happens in that case? does it throw any errors?

Reply:
Astra Pro recently moved to using window.scrollTo - see https://github.com/brainstormforce/astra-addon/pull/119

cc- @abhishek-bsf

Reply:
Updated @Nikschavan could you please?